web/handlers/version.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import os
+
+from git import Repo
```

Comment:
Do we need to add this to our dependencies?

Reply:
I think it's installed by default. It worked in my virtual env and in the kubernetes cluster with no installation.

Reply:
I added it to the requirements, just in case.